crates/metadata/src/tests.rs

```diff
@@ -389,3 +389,35 @@ fn construct_runtime_contract_spec() {
     );
     assert_eq!(event_spec, expected_event_spec);
 }
+
+/// Test for trimming in docs for event specification
+#[test]
+fn trim_events_in_specification() {
```

Comment:
This doesn't actually test the event docs, it is testing `ConstructorSpecBuilder::docs`